tests/cypress/e2e/features/issue_4662_Count_Annotations.js

```diff
@@ -0,0 +1,75 @@
+import { taskName, labelName } from '../../support/const';
```

Comment:
Please, rewrite with `cy.headlessCreateTask` as a newer approach. 
You may find example here: `tests/cypress/e2e/actions_tasks2/fit_image_different_res.js`